File: client/employee-management/src/app/module/employee-module/domain/Address.ts
export class Address {
  id: number;
  addressLine1: string;
  addressLine2: string;
  city: string;
  zipCode: string;
  stateName: string;
  stateAbbr: string;
  personId: number;


  constructor(id: number, addressLine1: string, addressLine2: string, city: string, zipCode: string, stateName: string, stateAbbr: string, personId: number) {
    this.id = id;
    this.addressLine1 = addressLine1;
    this.addressLine2 = addressLine2;
    this.city = city;
    this.zipCode = zipCode;
    this.stateName = stateName;
    this.stateAbbr = stateAbbr;
    this.personId = personId;
  }
}

File: server/src/main/java/com/example/server/domain/Address.java
package com.example.server.domain;

import lombok.*;

import javax.persistence.*;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Builder
@ToString
@Table(name = "address")
public class Address {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id", nullable = false)
    private int id;
    @Column(name = "address_line_1")
    private String addressLine1;
    @Column(name = "address_line_2")
    private String addressLine2;
    @Column(name = "city")
    private String city;
    @Column(name = "zipcode")
    private String zipCode;
    @Column(name = "state_name")
    private String stateName;
    @Column(name = "state_abbr")
    private String stateAbbr;
    @Column(name = "person_id")
    private int personId;
}
